app/controllers/katello/api/v2/distributions_controller.rb

```diff
@@ -1,38 +0,0 @@
-module Katello
-  class Api::V2::DistributionsController < Api::V2::ApiController
```

Comment:
I was suggesting we could maintain the existing controller, but instead of rendering distribution objects, we render repo objects with the distribution attributes.  But eric is now suggesting we just deprecate the distribution api in the 2.3 release (which is not out yet), and get rid of it in this PR